libcomposefs/lcfs-writer.c

```diff
@@ -568,17 +592,41 @@ struct lcfs_node_s *lcfs_load_node_from_file(int dirfd, const char *fname,
 	ret->inode.st_size = sb.st_size;
 
 	if ((sb.st_mode & S_IFMT) == S_IFREG) {
-		if (sb.st_size != 0 && (buildflags & LCFS_BUILD_COMPUTE_DIGEST) != 0) {
-			int fd = openat(dirfd, fname, O_RDONLY | O_CLOEXEC);
+		bool compute_digest = (buildflags & LCFS_BUILD_COMPUTE_DIGEST) != 0;
+		bool inline_small = (buildflags & LCFS_BUILD_INLINE_SMALL) != 0;
+		bool do_digest = sb.st_size != 0 && compute_digest;
+		bool do_inline = sb.st_size > 0 && sb.st_size <= 64 && inline_small;
```

Comment:
Let's hoist this `64` to a `static const` or a `#define` or something that's documented with at least a brief rationale.

Reply:
yeah